src/routes/user.tsx

```diff
@@ -170,8 +170,26 @@ const DeleteUserButton = ({ user }: { user: UserInfo }) => {
   )
 }
 
-const SetPasswordButton = ({ user }: { user: UserInfo }) => {
+const SetPasswordButton = ({
+  user,
+  self,
+  refetch,
+}: {
+  user: UserInfo
+  self: boolean
+  refetch: () => Promise<void>
+}) => {
   const emitError = useErrorEmitter()
+  const alertOnSaveAdmin = async () => {
+    if (self) return
+    const confirmation = await createDialog({
+      title: 'Confirm Edit',
+      description: 'Are you sure you want to make this change?',
```

Comment:
@varCepheid Can you tell me more about why we are sending an alert when you edit another user, but not yourself?

Reply:
That was my understanding of what we needed from this set of changes. If we want it to confirm for all users, I can change it.

Reply:
I think it would make more sense to me for it to either show the alert for all users or change the alert text to say something that makes it clear why it might be unexpected that you are editing another user (e.g. "Are you sure you want to edit user _____ (this is not your own account)" to prevent accidentally editing someone else when you meant to edit yourself).